runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -239,6 +252,7 @@ public void write(@Nonnull IToken token, @Nonnull Object data, @Nonnull CacheOpt
             // Cache the successful write
             if (cacheOption == CacheOption.WRITE_THROUGH) {
                 readCache.put(token.getSequence(), ld);
+                currentDataSizeInCache.addAndGet(ld.getSizeEstimate());
```

Comment:
This accounts for the the serialized size and not the in-memory size of the object.

```java

 @Override
    public int getSizeEstimate() {
        byte[] tempData = data;
        if (tempData != null) {
            return tempData.length;
        }

        if (lastKnownSize != NOT_KNOWN) {
            return lastKnownSize;
        }

        return 1;
    }
```

Reply:
https://github.com/CorfuDB/CorfuDB/blob/master/runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java#L197 is also calculating the size estimate, but before "serialization", so its most likely inaccurate. Remove it?